core/object/script_backtrace.cpp

```diff
@@ -52,6 +52,7 @@ void ScriptBacktrace::_store_variables(const List<String> &p_names, const List<V
 void ScriptBacktrace::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_language_name"), &ScriptBacktrace::get_language_name);
 
+	ClassDB::bind_method(D_METHOD("is_empty"), &ScriptBacktrace::is_empty);
```

Comment:
But why is this exposed now?

Reply:
Purely for convenience, to avoid having to do `backtrace.get_frame_count() == 0`.

I figured it might be handy.